apps/odyssey-stats/src/widget/mini-chart.jsx

```diff
@@ -66,23 +67,45 @@ const MiniChart = ( { siteId, quantity = 7, gmtOffset, odysseyStatsBaseUrl } ) =
 		return () => observer.disconnect();
 	} );
 
+	const isEmptyChart = ! chartData.some( ( bar ) => bar.value > 0 );
+	const placeholderChartData = Array.from( { length: 7 }, () => ( {
+		value: Math.random(),
+	} ) );
+
 	return (
 		<div
 			ref={ chartWrapperRef }
 			id="stats-widget-minichart"
 			className="stats-widget-minichart"
 			aria-hidden="true"
 		>
-			<div className="stats-widget-minichart__chart-head">
-				<Intervals selected={ period } compact={ false } onChange={ setPeriod } />
-			</div>
 			{ isLoading && <StatsModulePlaceholder className="is-chart" isLoading={ true } /> }
 			{ ! isLoading && (
 				<>
-					<Chart barClick={ barClick } data={ chartData } minBarWidth={ 35 }>
+					<div className="stats-widget-minichart__chart-head">
+						<Intervals selected={ period } compact={ false } onChange={ setPeriod } />
+					</div>
+					<Chart
+						barClick={ barClick }
+						data={ isEmptyChart ? placeholderChartData : chartData }
+						minBarWidth={ 35 }
+						isPlaceholder={ isEmptyChart }
+					>
 						<StatsEmptyState
-							infoText={ translate(
-								'There was no data recorded during the selected time period.'
+							headingText=""
+							infoText={ createInterpolateElement(
```

Comment:
We don't need to use `createInterpolateElement` here. Instead. we can use i18n-calypso's [markup support](https://github.com/Automattic/wp-calypso/tree/trunk/packages/i18n-calypso#mixing-strings-and-markup):

```js
const example = i18n.translate( 'I feel {{em}}very{{/em}} strongly about this.', {
	components: {
		em: <em />,
	},
} );
```